psi4/driver/json_wrapper.py

```diff
@@ -98,7 +98,6 @@ def run_json(json_data, clean=True):
     if return_output:
         with open(outfile, 'r') as f:
             json_data["raw_output"] = f.read()
-        os.unlink(outfile)
```

Comment:
What if you changed this to `atexit.register(os.unlink, outfile)`. You will need to `import atexit`.